pkg/kapis/iam/v1alpha2/handler.go

```diff
@@ -1099,10 +1099,10 @@ func (h *iamHandler) DescribeNamespaceRole(request *restful.Request, response *r
 
 // resolve the namespace which controlled by the devops project
 func (h *iamHandler) resolveNamespace(namespace string, devops string) (string, error) {
-	if devops == "" {
-		return namespace, nil
-	}
-	return h.am.GetDevOpsRelatedNamespace(devops)
+	//if devops == "" {
+	return namespace, nil
```

Comment:
Please pre-check if devops is empty, then return devops first if devops is not empty.

This fix could fix <https://github.com/kubesphere-sigs/ks-devops/issues/91>.

Test image: `johnniang/ks-apiserver:remove-devops-ctrl-iam`.

Reply:
Thanks for pointing this.